R/constants.R

```diff
@@ -190,6 +190,16 @@ aglu.MIN_SOIL_CARBON_DENSITY <- 0
 aglu.N0_LOGIT_EXP <- 0
 aglu.N0_LOGIT_TYPE <- NA
 
+# fraction of land that is protected
+aglu.PROTECT_LAND_FRACT <- 0.9
+
+# unManaged Land Value
+aglu.UNMANAGED_LAND_VALUE <- 1
```

Comment:
Yeah that's right, 1975$ per thousand square kilometers. This for land use regions with no commercial uses of land, so no ability to calculate average land value. I think there are 8 or 10 of them, mostly in the Arctic. Anyway it just needs to be a non-zero number so it doesn't blow up the land sharing equations.